extension.json

```diff
@@ -176,35 +176,66 @@
             "type": "string",
             "minLength": 1
           },
+          "identityMap": {
+            "type": "string",
+            "pattern": "^%[^%]+%$"
+          },
           "consent": {
             "anyOf": [
               {
                 "type": "array",
                 "minItems": 1,
-                "maxItems": 1,
                 "items": {
-                  "type": "object",
-                  "properties": {
-                    "standard": {
-                      "type": "string",
-                      "enum": ["Adobe"]
-                    },
-                    "version": {
-                      "type": "string",
-                      "enum": ["1.0"]
+                  "anyOf": [
+                    {
+                      "type": "object",
+                      "required": ["standard", "version", "value"],
```

Comment:
This `required` statement is not backward compatible. If we want to keep it (which I would like), we'll need to work with the Launch team to figure out if anyone has a settings object that does _not_ contain `standard`, `version`, or `value`.